lua/aerial/navigation.lua

```diff
@@ -237,6 +237,12 @@ M.select_symbol = function(item, winid, bufnr, opts)
     winid = vim.api.nvim_get_current_win()
     util.go_win_no_au(my_winid)
   end
+
+  if opts.jump then
+    local old_cursor = vim.api.nvim_win_get_cursor(winid)
+    vim.api.nvim_buf_set_mark(vim.api.nvim_get_current_buf(), [[']], unpack(old_cursor))
```

Comment:
I apologize for the terrible naming, this API needs some refactoring. The "jump" parameter doesn't mean what you think it means. When it's true, it means that we will focus the source window as part of this navigation action. So when you're just navigating around within the same buffer using `:AerialNext/Prev`, this value will be false.

Another thing to note: we're jumping to a location in `winid`, but `winid` is not necessarily the active window right here. You'll probably need to use `nvim_win_get_buf()` for the buffer, and you might need to do a `nvim_win_call` as well to make sure the mark is set in the appropriate window. If you want to test it, I would expect this to come up if your cursor is in the aerial window when you run `:AerialNext/Prev`.